common/djangoapps/third_party_auth/samlproviderconfig/serializers.py

```diff
@@ -13,6 +13,12 @@ class SAMLProviderConfigSerializer(serializers.ModelSerializer):  # lint-amnesty
     class Meta:
         model = SAMLProviderConfig
         fields = '__all__'
+        validators = [
```

Comment:
what is the purpose of this relative to the validation done in the `save()` override in the model?

Reply:
this is the view level validation to add as a second level of safeguarding to the system. The system will throw an integrity error from the DB level if a record generation is attempted with existing entity ID's but I also wanted to add validation on the api view level as well.